cmd/vpu_plugin/README.md

```diff
@@ -1,75 +1,240 @@
-device-plugin start server at: /var/lib/kubelet/device-plugins/vpu.intel.com-hddl.sock
-device-plugin registered
-```
+### Build the plugin image
 
-### Deploy VPU device plugin as a DaemonSet:
+The following will use `docker` to build a local container image called
+`intel/intel-vpu-plugin` with the tag `devel`.
 
-#### Build plugin image
-```
+The image build tool can be changed from the default `docker` by setting the `BUILDER` argument
+to the [`Makefile`](Makefile).
+
+```bash
+$ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
 $ make intel-vpu-plugin
+...
+Successfully tagged intel/intel-vpu-plugin:devel
 ```
 
-#### Create plugin DaemonSet
-```
+### Deploy plugin DaemonSet
+
+You can then use the example DaemonSet YAML file provided to deploy the plugin.
+
+```bash
 $ kubectl create -f ./deployments/vpu_plugin/vpu_plugin.yaml
 daemonset.apps/intel-vpu-plugin created
 ```
-**Note**: It is also possible to run the VPU device plugin using a non-root user. To do this,
-the nodes' DAC rules must be configured to allow USB device descriptor detection, device plugin
-socket creation and kubelet registration. Furthermore, the deployments `securityContext` must
-be configured with appropriate `runAsUser/runAsGroup`.
 
-### Verify VPU device plugin is registered on master:
+> **Note**: It is also possible to run the VPU device plugin using a non-root user. To do this,
+the nodes' DAC rules must be configured to device plugin socket creation and kubelet registration.
+Furthermore, the deployments `securityContext` must be configured with appropriate `runAsUser/runAsGroup`.
+
+## Deploy by hand
+
+For development purposes, it is sometimes convenient to deploy the plugin 'by hand' on a node.
+In this case, you do not need to build the complete container image, and can build just the plugin.
+
+### Build the plugin
+
+First we build the plugin:
+
+```bash
+$ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
+$ make vpu_plugin
+```
+
+### Run the plugin as administrator
+
+Now we can run the plugin directly on the node:
+
+```bash
+$ sudo $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes/cmd/vpu_plugin/vpu_plugin
+VPU device plugin started
 ```
-$ kubectl describe node <node name> | grep vpu.intel.com
- vpu.intel.com/hddl:  1
- vpu.intel.com/hddl:  1
+
+## Verify plugin registration
+
+You can verify the plugin has been registered with the expected nodes by searching for the relevant
+resource allocation status on the nodes:
+
+```bash
+$ kubectl get nodes -o=jsonpath="{range .items[*]}{.metadata.name}{'\n'}{' hddl: '}{.status.allocatable.vpu\.intel\.com/hddl}{'\n'}"
+vcaanode00
+ hddl: 12
 ```
 
-### Test VPU device plugin:
+## Testing the plugin
+
+We can test the plugin is working by deploying the provided example OpenVINO image with HDDL plugin enabled.
+
+1. Build a Docker image with an classification example:
```

Comment:
just a nitpick - I think I changed the numbered lists into titled sections in the other docs, as the numbered lists and thus indented code and words are a bit more painful to maintain in markdown - not a blocker, just a note.

Reply:
done